### PR TITLE
Updated missing Docs for ui hooks

### PR DESCRIPTION
**Pull request by gautamsi**
some minimal docs for ui hooks
also fixed example in one of the merged changeset.

### CODE REVIEW

**Comment by gautamsi** [comment]
I have to add document for #2483 adding which now will cause the conflict. once either is merged, I will fix the docs.

File: packages/app-admin-ui/README.md [hunk 1]
```diff
@@ -72,14 +72,59 @@ export default {
...
+```
+
+#### `listHeaderActions`
+
+Header components on the List page can be replaced using this hook. This replaces components on list page for all Lists.
+
+> This must return a React component.
+
+```javascript
+// ./admin-ui/index.js
```
Comment (Vultraz) [inline packages/app-admin-ui/README.md:105]: Should use the `title=./admin-ui/index.js` syntax on the line above.

Comment (gautamsi) [reply packages/app-admin-ui/README.md]: i did that but what advantage it has?

Comment (Vultraz) [reply packages/app-admin-ui/README.md]: image

Comment (gautamsi) [reply packages/app-admin-ui/README.md]: thought so but the rendering does not work. does leading `.` matter in file name?

Comment (gautamsi) [reply packages/app-admin-ui/README.md]: or is it thing for rendering in keystonejs.com?

Comment (Vultraz) [reply packages/app-admin-ui/README.md]: Perhaps it's the leading `.` 🤔

Comment (gautamsi) [reply packages/app-admin-ui/README.md]: I see that it is rendering on docs website

**Pull request merged**